skimage/util/_coord_conv.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+
+
+def xy_to_rc(points=None, image=None):
+    """Convert points' coordinates and/or image from `x, y` to `r, c` convention.
+
+        o--->....       o--->....
+        |   x   .       |   c   .
+        |       .  ==>  |       .
+        v y     .       v r     .
+        .........       .........
+
+    Parameters
+    ----------
+    points : (K, 2) ndarray, optional
```

Comment:
@soupault http://scikit-image.org/docs/dev/user_guide/numpy_images.html#coordinate-conventions =)

The convention is (pln, row, col) or (p, r, c) for NumPy coordinates, (x, y, z) for xy/cartesian. In general, NumPy and xy axis order is reversed. For Cartesian, I'm not sure but I think only y needs to be inverted.